invenio_records_rest/views.py

```diff
@@ -117,6 +117,7 @@ def create_url_rules(endpoint, list_route=None, item_route=None,
                      delete_permission_factory_imp=None,
```

Comment:
Can you fix the copyright year in this file please?

Reply:
Absolutely, done. Sorry I didn't notice it here.
I have also added the release commit.